sources/postgres/data.go

```diff
@@ -132,6 +132,8 @@ func convScalar(spannerType ddl.Type, srcTypeName string, location *time.Locatio
 		return val, nil
 	case ddl.Timestamp:
 		return convTimestamp(srcTypeName, location, val)
+	case ddl.Json:
+		return val, nil
```

Comment:
Should we wrap this in a more specific type e.g., NullJSON - https://github.com/googleapis/google-cloud-go/blob/4537b45d2611ce480abfb5d186b59e7258ec872c/spanner/value.go#L774 ?